Sources/AirRobeWidget/Views/OptInView/AirRobeOptInViewModel.swift

```diff
@@ -48,6 +48,12 @@ final class AirRobeOptInViewModel {
             return
         }
 
+        if let testVariant = shoppingDataModel.getTargetSplitTestVariant(),
+           testVariant.targetSplitTestVariant == AirRobeStrings.airrobeDisabled && !testVariant.enabled {
+            isAllSet = .widgetDisabled
+            return
+        }
```

Comment:
The targetSplitTestVariant property should not be checked.

I suspect we might need something like:

```
if let testVariant = shoppingDataModel.getTargetSplitTestVariant(),
           !testVariant.enabled {
            isAllSet = .widgetDisabled
            return
        }
``` ?

Reply:
🙌